### ISSUE TITLE
fix the `npm test` script

### ISSUE THREAD

**Issue by theRemix** [edsperanto/devleague--behavior-driven-shopping-list#10]
```
  "scripts": {
    "test": "echo 'run http-server or live-server then open http://localhost:8080/tests.html to run tests!'"
  },
```

**Comment by ed-scaped** [comment]
fixed

**Comment by theRemix** [comment]
for future reference, on the commit that fixes this issue, in the commit message use https://help.github.com/articles/closing-issues-via-commit-messages/

```
[your normal commit message]

resolves #10
```
